include/vapor/OpenMPSupport.h

```diff
@@ -1,13 +1,14 @@
-#pragma once
+#ifndef OPENMPSUPPORT_H
```

Comment:
Just curious. Is there a problem with using "#pragma once"? It's used elsewhere in the codebase.

Reply:
It was giving me "multiple definition" errors.  I suspect that `#pragma once` was not helping at least.  (Commit 60dba6057f83f7d4e29aeb32fa2fa565c8070d9a finally fixed the compilation errors.)

Reply:
If using include guards rather than pragma once fixes the issue the real problem is likely still lurking underneath.

Reply:
Maybe I wasn't clear enough. My guess is that `#pragma once` is innocent for errors I run into earlier. @StasJ  rest assured: any lurking bug will be dug out and thrown away ;)